pkg/kubelet/cm/helpers_linux.go

```diff
@@ -44,3 +48,36 @@ func GetCgroupSubsystems() (*CgroupSubsystems, error) {
 		MountPoints: mountPoints,
 	}, nil
 }
+
+// readProcsFile takes a cgroup directory name as an argument
+// reads through the cgroup's procs file and returns a list of tgid's.
+// It returns an empty list if a procs file doesn't exists
+func readProcsFile(dir string) ([]int, error) {
+	procsFile := filepath.Join(dir, "cgroup.procs")
```

Comment:
Why not use `tasks` instead? I think `tasks` includes threads, whereas, procs does not. can you verify that?